tests/bullet/configure

```diff
@@ -3213,158 +3213,6 @@ echo "${ECHO_T}$ac_cv_c_compiler_gnu" >&6; }
 GCC=`test $ac_compiler_gnu = yes && echo yes`
```

Comment:
What is this change?

Reply:
Bullet's configure file automatically adds `-g` to the Makefiles it generates. That ends up crashing due to https://bugs.llvm.org/show_bug.cgi?id=44920 so I removed the relevant configure script portions to avoid that. (This will also make it faster to build and link, but we do lose some coverage of debug info generation and linking.)

Reply:
Ick.   configure scripts are supposed to autogenerated really.    Hacking it like this seems like a bad idea.  Since we already test the actually bullet code via cmake anyway, how about just disabling the configure version of `get_bullet_library` in `test_core.py` with a TODO to re-enable once we fix the llvm bug?

We already skip the autoconf version of this test on windows anyway.

Reply:
Sounds good, thanks, doing it that way.